src/cli/commands/mod.rs

```diff
@@ -15,19 +13,14 @@ pub enum Commands {
 
 /// Bahaviour of a command
```

Comment:
typo

Reply:
you mean Behavior ? we need to choose between US and GB :-)

Reply:
Oh, even when re-reading it I missed it ^^
It is fixed